mqtt-streaming/src/test/scala/docs/scaladsl/MqttSessionSpec.scala

```diff
@@ -167,6 +167,51 @@ class MqttSessionSpec
       result.futureValue shouldBe Right(Event(subAck))
     }
 
+    "Subscribe and stash any subsequent subscriptions" in {
```

Comment:
Additional test - nothing directly to do with the problem being solved by the PR